coretex/utils/docker.py

```diff
@@ -53,6 +53,20 @@ def removeNetwork(name: str) -> None:
     command(["docker", "network", "rm", name], ignoreStdout = True, ignoreStderr = True)
 
 
+def removeImage(image: str) -> None:
+    command(["docker", "image", "rm", image], ignoreStdout = True, ignoreStderr = True)
+
+
+def removeDanglingImages(repository: str) -> None:
+    _, output, _ = command(["docker", "image", "ls", "--format", "json"], ignoreStdout = True, ignoreStderr = True)
+    images = output.strip().split("\n")
+    for image in images:
+        if len(image) > 0:
+            jsonImg = json.loads(image)
+            if jsonImg["Tag"] == "<none>":
```

Comment:
You can pass in the currently used tag as a parameter to this function, and then remove all images which do not have that tag.